lib/src/checkers/minimum_tap_area_checker.dart

```diff
@@ -4,6 +4,52 @@ import 'package:flutter/rendering.dart';
 import '../accessibility_issue.dart';
 import 'checker_base.dart';
 
+/// Defines the minimum tap size per device type.
+class MinimumTapAreas {
+  const MinimumTapAreas({
+    required this.mobile,
+    required this.desktop,
+  });
+
+  /// The minimum tap areas as defined by the Material Design guidelines.
+  ///
+  /// 48 on mobile devices with touch screens, and 28 on desktop devices.
```

Comment:
Hmm, would it be useful to have link to source of these.

Reply:
Good idea, added